web-client/src/presenter/state-public.ts

```diff
@@ -27,7 +27,9 @@ const computeds = {
     typeof practitionerSearchHelper
   >,
   publicCaseDetailHeaderHelper,
-  publicCaseDetailHelper,
+  publicCaseDetailHelper: publicCaseDetailHelper as unknown as ReturnType<
+    typeof publicCaseDetailHelper
+  >,
```

Comment:
Is the type not inferred here properly without the `as unknown as ReturnType<...`?

Reply:
Thank you!

Reply:
No, helper is a function and we want to type it as the results of the function.